packages/trader/src/Stores/ui-store.js

```diff
@@ -13,15 +13,17 @@ import { sortNotifications } from '../App/Components/Elements/NotificationMessag
 const store_name = 'ui_store';
 
 export default class UIStore extends BaseStore {
-    @observable is_main_drawer_on          = false;
-    @observable is_notifications_drawer_on = false;
-    @observable is_positions_drawer_on     = false;
-    @observable is_reports_visible         = false;
+    @observable is_main_drawer_on           = false;
+    @observable is_notifications_drawer_on  = false;
+    @observable is_positions_drawer_on      = false;
+    @observable is_account_settings_visible = false;
+    @observable is_reports_visible          = false;
 
     @observable is_cashier_modal_on     = false;
     @observable is_dark_mode_on         = false;
     @observable is_settings_modal_on    = false;
     @observable is_accounts_switcher_on = false;
+    @observable is_account_management_modal_on = false;
```

Comment:
We can remove this since we're using a page route modal.

Reply:
yea this was what i was asking just now